.vscode/setup-scripts.code-snippets

```diff
@@ -0,0 +1,17 @@
+{
+  "Set up a react component": {
+    "prefix": "setup-react",
+    "scope": "javascript",
+    "body": [
+      "import React from \"react\";",
+      "",
+      "const $TM_FILENAME_BASE = () => {",
```

Comment:
this sets the const name as the filename base. so if the file is named Blah.js, this would equal Blah.

You can alternatively set this to a $n where n is a number, and the cursor goes to each spot in numerical order. So if you have a `console.log($1)`, and in another spot a `console.log($2)` the cursor would first go to the $1, and then to the $2 after you finish typing and hit tab. 

You can use the same number in multiple spots, and it would have the cursor selected on both spots. For example, instead of the $TM_FILENAME_BASE on this line and line 13, you could have a $1 and both would be automatically selected for you to type out the const name yourself.